typescript-forms/form-events.ts

```diff
@@ -0,0 +1,37 @@
+function handleFocus(event: Event): void {
+  console.log('The focus event has fired');
+  const eventTarget = event.target as HTMLElement | HTMLTextAreaElement | any;
+  console.log('eventTarget.name:', eventTarget.name);
+}
+
+function handleBlur(event: Event): void {
+  console.log('the blur event has fire');
+  const eventTarget = event.target as HTMLElement | HTMLTextAreaElement | any;
+  console.log('eventTarget.name:', eventTarget.name);
+}
+
+function handleInput(event: Event): void {
+  const eventTarget = event.target as HTMLElement | HTMLTextAreaElement | any;
```

Comment:
You do not want to give it a type of `any`. That will defeat the purpose of assigning a type in the first place